source/common/http/null_route_impl.h

```diff
@@ -198,7 +189,7 @@ struct RouteEntryImpl : public Router::RouteEntry {
   const Router::EarlyDataPolicy& earlyDataPolicy() const override { return *early_data_policy_; }
 
   std::unique_ptr<const HashPolicyImpl> hash_policy_;
-  std::unique_ptr<Router::RetryPolicy> retry_policy_;
+  const Router::RetryPolicy* retry_policy_;
```

Comment:
```suggestion
  const Router::RetryPolicy& retry_policy_;
```

Per Envoy style, please use reference.

Reply:
done